docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1109 @@
+
+  3. **Submit Resolve Request**: Call the **Trust Anchor**'s ``/resolve`` endpoint (as defined in :ref:`trust:Trust Infrastructure Requirements`) with URL-encoded parameters:
+
+    - ``sub``: Federation Entity Identifier.
+    - ``trust_anchor``: **Trust Anchor** Federation Entity Identifier (always the root Trust Anchor, even for Intermediate-mediated onboarding).
+
+    Example resolve request:
+
+    .. code-block:: http
+
+        GET /resolve?sub=https%3A%2F%2Fcredentials.example.gov&trust_anchor=https%3A%2F%2Ftrust-anchor.example.gov HTTP/1.1
+        Host: trust-anchor.example.gov
+
+**Step 4: Resolve Response and Onboarding Completion**
+
+Following the resolve request, the **Federation Authority** performs:
+
+  - **Trust Chain Reconstruction**: Reconstruction of a valid Trust Chain for the entity as defined in :ref:`trust:The Infrastructure of Trust`.
+  - **Federation Trust Mark Generation**: Generation of an IT-Wallet Federation Trust Mark as a signed JWT attestation of the entity's federation membership and compliance with IT-Wallet technical requirements.
+  - **Trust Mark Integration in Subordinate Statement**: The generated Trust Mark is included in the entity's Subordinate Statement as defined in :ref:`trust:Subordinate Statements`.
+  - **Metadata Policy Application**: Application of cascading metadata policies during Trust Chain construction, where Trust Anchor policies take precedence over Intermediate policies.
+  - Generation of a signed JSON Web Token (JWT) using algorithms specified in :ref:`algorithms:Cryptographic Algorithms` containing the reconstructed Trust Chain and validated entity metadata.
+  - Transmission of an HTTP response containing the created JWT (Resolve Response).
+
+If the response status code is ``200 OK``, the Federation Entity MUST complete the onboarding process by:
+
+  - **Validate Resolve Response**: Validate the JWT contained in the Resolve Response and extract the Trust Chain and validated metadata from the JWT payload.
+  - **Fetch Subordinate Statement**: Retrieve its own Subordinate Statement from the immediate Federation Authority using the ``/fetch`` endpoint as defined in :ref:`trust:Federation API endpoints`.
+  - **Extract Trust Mark**: Extract the Federation Trust Mark from the Subordinate Statement ``trust_marks`` claim.
+  - **Trust Mark Integration**: Include the extracted Trust Mark in its Entity Configuration using the ``trust_marks`` claim as specified in :ref:`trust:Entity Configuration Leaves and Intermediates`.
+  - **Final Entity Configuration Update**: Publish the updated Entity Configuration with the integrated Trust Mark at the ``/.well-known/openid-federation`` endpoint.
+
+Upon successful completion of Step 4, the **entity onboarding is successfully completed**. The entity is now operational within the IT-Wallet federation and ready for operational activities.
```

Comment:
this step must be replaced with a request to the fetch endpoint, that's enough to prove the succesful onboarding.